qdata/parser.py

```diff
@@ -287,16 +385,32 @@ def uop(self, *args):
         return Gate(op_name, params, wires)
 
     def anylist(self, *args):
-        # either <idlist> or <mixedlist>
+        """Either ``idlist`` or ``mixedlist``"""
```

Comment:
I like how the grammar is mirrored in the above functions directly with a text block. Would be nice to do that with `anylist`, `idlist`, `argument`, and `explist` too

Reply:
Will do this now 👍

Reply:
done!